.github/workflows/acceptance.yml

```diff
@@ -147,7 +147,8 @@ jobs:
     strategy:
       matrix:
         instance: [0, 1, 2, 3]
-
+        # TODO(joshuarli): Add 3.8.12.
+        python-version: [3.6.13]
```

Comment:
I think this might mess up the freight stuff since the check titles change as a result? Will have to take a look at it before merging, I might just modify it to ignore stuff enclosed in `()` or something.

Reply:
Oh well, this is a getsentry concern, so doesn't block this PR, but what about the cross repo machinery @billyvg?

Reply:
@joshuarli hmm, it shouldn't be affected for cross-repo tasks, but yeah it will affect freight if we do the same things in getsentry.